airdrop/alembic/versions/7f844b08e3c9_rename_columns.py

```diff
@@ -0,0 +1,34 @@
+"""rename_columns
+
+Revision ID: 7f844b08e3c9
+Revises: 384051a9013e
+Create Date: 2022-07-01 17:13:09.819412
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = '7f844b08e3c9'
+down_revision = '384051a9013e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
These columns are dropping and then creating instead renaming.

Reply:
Resolved